Dockerfile

```
@@ -0,0 +1,31 @@
+FROM ubuntu
+
+SHELL [ "/bin/bash", "-c" ]
+
+WORKDIR /home/ubuntu
+
+COPY packages ./packages
+
+COPY requirements.txt ./
+
+RUN apt update
```

Hadolint (2.12.0)

[warning] Do not use apt as it is meant to be a end-user tool, use apt-get or apt-cache instead

(DL3027)